linera-views/src/backends/lru_caching.rs

```diff
@@ -145,7 +186,13 @@ impl LruPrefixCache {
     /// create new entries in the cache.
     pub fn delete_prefix(&mut self, key_prefix: &[u8]) {
         if self.has_exclusive_access {
-            for (_, value) in self.map.range_mut(get_interval(key_prefix.to_vec())) {
+            for (key, value) in self.map.range_mut(get_interval(key_prefix.to_vec())) {
+                *self.queue.get_mut(key).unwrap() = key.len();
+                let value_size = match value {
+                    CacheEntry::Value(vec) => vec.len(),
+                    _ => 0,
+                };
```

Comment:
It looks like this `match { .. }` should be method of `CacheEntry`

Reply:
Ok, done.